### PR TITLE
🚧Change release todo issue templates

### PR DESCRIPTION
**Pull request by kselena**
<h1 style="font-weight:bold" align="center">Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">📋Review Checklist📋</h2>

- [x] The **_head(source)_** branch for this pull request is a **_feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: feature/<issue-num>-<description>
      Example: feature/123-my-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_development_** branch with the name **_develop_**.

- [x] The **_base(target)_** branch for this pull request is a **_development_** branch with the name **_develop_**.
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

💡For more information on branching, refer to the project's [branching documentation](../../Documentation/Branching.md).

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] Pull request title matches the title of the associated issue.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] Project assigned matches the project of the associated issue.

- [x] Milestone assigned matches the milestone of the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out, and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_develop_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/preview-release-todo-issue-template.yml [hunk 1]
```diff
@@ -2,26 +2,13 @@ name: 🚀 Preview Release To Do List
...
-
 
 - type: checkboxes
   attributes:
     label: Pre-Release ToDo List
     description: Complete the following items to perform a release.
     options:
-    - label: All issues listed in the **_Issues Released_** section above.
-    - label: All issues in the _**Issues Released**_ section are closed and all issue tasks are complete.
+    - label: All issues in the current milestone are closed and all issue tasks are complete.
```
Comment (KinsonDigitalAdmin) [inline .github/ISSUE_TEMPLATE/preview-release-todo-issue-template.yml:11]: This should be like the value below.
Just like it was called out in the issue.

I think there was some miscommunication.  Do you think that it is better for having **_current_** instead of **_assigned_**?

```suggestion
    - label: All issues in the assigned milestone are closed and all issue tasks are complete.
```

File: .github/ISSUE_TEMPLATE/production-release-todo-issue-template.yml [hunk 2]
```diff
@@ -2,26 +2,13 @@ name: 🚀 Production Release To Do List
...
-
 
 - type: checkboxes
   attributes:
     label: Pre-Release ToDo List
     description: Complete the following items to perform a release.
     options:
-    - label: All issues listed in the **_Issues Released_** section above.
-    - label: All issues in the _**Issues Released**_ section are closed and all issue tasks are complete.
+    - label: All issues in the current milestone are closed and all issue tasks are complete.
```
Comment (KinsonDigitalAdmin) [inline .github/ISSUE_TEMPLATE/production-release-todo-issue-template.yml:11]: This should be like the value below.
Just like it was called out in the issue.

I think there was some miscommunication.  Do you think that it is better for having **_current_** instead of **_assigned_**?

```suggestion
    - label: All issues in the assigned milestone are closed and all issue tasks are complete.
```

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/preview-release-todo-issue-template.yml (hunk 1 above)
Comment (kselena) [reply .github/ISSUE_TEMPLATE/preview-release-todo-issue-template.yml:11]: For consistency, I think assigned is good, but is it assigned or chosen?  If it's chosen, then I'd put "current" instead of "assigned."

File: .github/ISSUE_TEMPLATE/production-release-todo-issue-template.yml (hunk 2 above)
Comment (kselena) [reply .github/ISSUE_TEMPLATE/production-release-todo-issue-template.yml:11]: For consistency, I think assigned is good, but is it assigned or chosen?  If it's chosen, then I'd put "current" instead of "assigned."

File: .github/ISSUE_TEMPLATE/preview-release-todo-issue-template.yml (hunk 1 above)
Comment (KinsonDigitalAdmin) [reply .github/ISSUE_TEMPLATE/preview-release-todo-issue-template.yml:11]: I would say assigned because all of the issues and PRs are literally **ASSIGNED** to milestones.

**Review by kselena — COMMENTED** [review]
Looks good.  Approved.

File: .github/ISSUE_TEMPLATE/production-release-todo-issue-template.yml (hunk 2 above)
Comment (KinsonDigitalAdmin) [reply .github/ISSUE_TEMPLATE/production-release-todo-issue-template.yml:11]: I would say assigned because all of the issues and PRs are literally ASSIGNED to milestones.

**Review by KinsonDigitalAdmin — APPROVED** [review]
